images/tidb-backup-manager/Dockerfile

```diff
@@ -20,9 +20,9 @@ ARG TOOLKIT_PKG_NAME=tidb-community-toolkit-${TOOLKIT_NAME}
 RUN \
   wget -nv https://download.pingcap.org/${TOOLKIT_PKG_NAME}.tar.gz \
   && tar -xzf ${TOOLKIT_PKG_NAME}.tar.gz \
-  && tar -xzf ${TOOLKIT_PKG_NAME}/tidb-lightning-${TOOLKIT_NAME}.tar.gz && mv tidb-lightning /tidb-lightning \
```

Comment:
why remove `tidb-lightning` and `dumpling`? I remember that `tidb-backup-manager` contains functions of them.

Reply:
`tar -xzf ${TOOLKIT_PKG_NAME}/tidb-lightning-${TOOLKIT_NAME}.tar.gz ` already unzip `tidb-lightning` to current dir. the old scripts will rasie an error 

```
mv: 'dumpling' and '/dumpling' are the same file
```
see more in  https://github.com/pingcap/tidb-operator/actions/runs/4487161238/jobs/7890311242